alpha/scripts/utils/setEntryProperties.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+if($argc < 3)
+{
+    echo "Arguments missing.\n\n";
+    echo "Usage: php " . __FILE__ . " {mapping} {isDateString} <realrun / dryrun> \n";
+    exit;
+}
+$mapping = $argv[1];
+$isDateString = ($argv[2] === "true");
+$dryRun = ($argv[3] === "dryrun");
+
+require_once(__DIR__ . '/../bootstrap.php');
+
+
+KalturaStatement::setDryRun($dryRun);
+
+$entryMappings = file($mapping, FILE_IGNORE_NEW_LINES);
+
+$properties = [""];
+$counter = 0;
+entry::setAllowOverrideReadOnlyFields(true);
+foreach ($entryMappings as $entryMapping)
+{
+    if($counter++ == 0){
+        $entryColumns = explode(",", $entryMapping);
+        foreach($entryColumns as $property){
+            if($property == 'id' || $property == 'entryId'){
+                continue;
+            }
+            if(!method_exists('entry', "set".ucfirst($property))){
+                echo "Property $property does not have the set function \n";
+                continue;
+            }
+            $properties[] = $property;
+        }
+        continue;
+    }
+    $entryValues= explode(",", $entryMapping);
+
+    $z = 0;
+    foreach($entryValues as $entryValue){
+        if($z === 0){
```

Comment:
@TalDubovKaltura  This approach is very hard to follow. Instead I suggest that doing
`entryValues = array_combine($properties,$entryValues)` 
and then you can just call 
`$entry = entryPeer::retrieveByPK($entryValues['id']);`